src/components/MainView/getDashArrayFromDb.js

```diff
@@ -0,0 +1,12 @@
+export default (snapVal) => {
```

Comment:
This entire file is redundant and not present in the development branch at the time. No need to restore it